data/templates/src/elf/dll/elf_dll_aarch64_template.s

```diff
@@ -88,6 +88,7 @@ strtab:
  db 0
  db 0
 strtabsz equ $ - strtab
+
+align 16
```

Comment:
We haven't been able to work out definitively if this should be aligned to 4, 8, or 16 - but it definitely needs to be aligned to work